recipes-kernel/linux/linux-nezha-dev.bb

```diff
@@ -0,0 +1,22 @@
+require recipes-kernel/linux/linux-mainline-common.inc
+FILESEXTRAPATHS:prepend = "${FILE_DIRNAME}/linux-nezha:"
+SUMMARY = "Nezha dev kernel recipe"
+
+KERNEL_VERSION_SANITY_SKIP = "1"
+
+SRCREV = "af3f4a1caec12845b809fba959e6334ab3b52a40"
+FORK ?= "tekkamanninja"
+BRANCH ?= "allwinner_nezha_d1_devel"
+SRC_URI = "git://github.com/${FORK}/linux.git;protocol=https;branch=${BRANCH} \
+           file://0001-riscv-fix-build-with-binutils-2.38.patch \
+           file://enable-autofs4.cfg \
+           file://enable-cgroups.cfg \
+           file://enable-modules.cfg \
```

Comment:
instead of cfg files can we use the KERNEL_FEATURES variable as much as possible ? see
https://git.yoctoproject.org/yocto-kernel-cache/tree/
for possible options.

Reply:
Okey, I didn't know about it earlier. It is replaced with this:
```
KERNEL_FEATURES += "features/cgroups/cgroups.cfg"
KERNEL_FEATURES += "ktypes/standard/standard.cfg"
```